srcpkgs/monero-gui/template

```diff
@@ -22,17 +22,17 @@ license="BSD-3-Clause"
 homepage="https://getmonero.org"
 distfiles="https://github.com/monero-project/monero-gui/archive/v${version}.tar.gz
  https://github.com/monero-project/monero/archive/v${_monero_version}.tar.gz>monero-${_monero_version}.tar.gz
- https://github.com/tevador/RandomX/archive/v${_randomx_version}.tar.gz
+ https://github.com/tevador/RandomX/archive/${_randomx_gitrev}.tar.gz
  https://github.com/Tencent/rapidjson/archive/${_rapidjson_gitrev}.tar.gz
  https://github.com/monero-project/supercop/archive/${_supercop_gitrev}.tar.gz
  https://github.com/dlbeer/quirc/archive/${_quirc_gitrev}.tar.gz"
```

Comment:
```suggestion
 https://github.com/tevador/RandomX/archive/${_randomx_gitrev}.tar.gz>randomx-${_randomx_gitrev}.tar.gz
 https://github.com/Tencent/rapidjson/archive/${_rapidjson_gitrev}.tar.gz>rapidjson-${_rapidjson_gitrev}.tar.gz
 https://github.com/monero-project/supercop/archive/${_supercop_gitrev}.tar.gz>supercop-${_supercop_gitrev}.tar.gz
 https://github.com/dlbeer/quirc/archive/${_quirc_gitrev}.tar.gz>quirc-${_quirc_gitrev}.tar.gz"
```
and I would do the same in the monero package and for monero-gui.

Reply:
Not sure what you mean here. https://github.com/Tencent/rapidjson/archive/rapidjson-129d19ba7f496df5e33658527a7158c79b99c21c.tar.gz would be a broken link/404?

Reply:
Understood and all occurances adapted accordingly.